feature/search/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/search/SearchScreen.kt

```diff
@@ -298,9 +298,11 @@ private fun SearchResultBody(
     searchQuery: String = "",
 ) {
     val state = rememberLazyStaggeredGridState()
+    val itemsAvailable by remember {
```

Comment:
This won't do what you may think it does. The `remember` isn't keyed on topics or news resources. Also the variables aren't backed by compose state so this change introduces a bug.